scss/components/_case-list-panel.scss

```diff
@@ -4,7 +4,8 @@
   padding: 5px;
   text-align: center;
 
-  .fa.fa-big {
+  .fa.fa-big,
+  .material-icons {
     color: $crm-gray-matte;
     font-size: 64px;
```

Comment:
You are defining this `$font-size-large-icon` variable, but you are not using it.